tests/PublicVisibility/Client/Updater.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace Tuf\Tests\PublicVisibility\Client;
+
+class Updater extends \Tuf\Client\Updater
```

Comment:
To get rid of this class and future tests classes like this I think it is simpler to do

`$method = new \ReflectionMethod(Updater::class, 'checkRollbackAttack');
  $method->setAccessible(true);
  $method->invoke($sut, $localMetadata, $incomingMetadata);`

This makes it very explicit that testing a non-public method in the test itself.